mscore/fotomode.cpp

```diff
@@ -20,7 +20,7 @@
 #include "libmscore/image.h"
 #include "libmscore/mscore.h"
 #include "svggenerator.h"
-#include "view/widgets/inspectordockwidget.h"
+#include "mu4/scenes/inspector/view/widgets/inspectordockwidget.h"
```

Comment:
It is better to transfer this class to `mscore`

Reply:
firstly, I wanted to keep all the "inspector" stuff in the "inspector" module, but since I've moved the adapter into mscore module, I can agree with that

Reply:
done